bootstrap/kubeadm/internal/cloudinit/controlplane_join.go

```diff
@@ -43,9 +47,10 @@ runcmd:
 type ControlPlaneJoinInput struct {
 	BaseUserData
 	secret.Certificates
-
-	BootstrapToken    string
-	JoinConfiguration string
+	UseExperimentalRetryJoin bool
```

Comment:
```suggestion
	UseExperimentalRetry bool
```

Given that the struct is already focused on Join